python/pyspark/sql/readwriter.py

```diff
@@ -234,6 +234,8 @@ def json(self, path, schema=None, primitivesAsString=None, prefersDecimal=None,
                                 default value, ``yyyy-MM-dd'T'HH:mm:ss.SSSXXX``.
         :param multiLine: parse one record, which may span multiple lines, per file. If None is
                           set, it uses the default value, ``false``.
+        :param lineSep: defines the line separator that should be used for parsing. If None is
+                        set, it uses the default value, ``\\r\\n`` or ``\\n``.
```

Comment:
seems the default value is always `\n`? https://github.com/apache/spark/pull/18581/files#diff-059fbd7487f6bec7cbd8a57f21f8c8c5R48